src/box/lua/load_cfg.lua

```diff
@@ -187,6 +188,56 @@ local default_cfg = {
     txn_isolation         = "best-effort",
 }
 
+-- We need to track cfg changes done through API of distinct modules (log.cfg of
+-- log module for example). We cannot use just box.cfg because it is not
+-- available before box.cfg() call and other modules can be configured before
+-- this moment.
+local pre_load_cfg = table.copy(default_cfg)
+
+-- On first box.cfg{} we need to know options that were already configured
+-- in standalone modules (like log module). We should not apply env vars
+-- for these options. pre_load_cfg is not suitable for this purpose because
+-- of nil values.
+local pre_load_cfg_is_set = {}
+
+local replication_sync_timeout_brief = [[
+Sets the default value for box.cfg.replication_sync_timeout.
+Old is 300 seconds, new is 0 seconds. New behaviour makes
+box.cfg{replication = ...} call exit without waiting for
+synchronisation with all the remote nodes. This means that the node
+might be in 'orphan' state for some time after the box.cfg{} call
+returns.
+
+Set before first box.cfg{} call in order for the option to take effect.
+]]
```

Comment:
Please, add https://github.com/tarantool/tarantool/wiki/compat%3Areplication_sync_timeout link to end of the brief description (see `src/lua/compat.lua` for examples) and add it into the list of the options: https://github.com/tarantool/tarantool/wiki/compat

Reply:
Done. Thanks!